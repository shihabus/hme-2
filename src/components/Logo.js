import React from "react";
import logo from "../assets/logo.png";
import styled from "styled-components";
import PropTypes from "prop-types";

const Image = styled.img`
  width: ${props => props.logoWidth};
  height: auto;
  align-self: center;
`;

export default function Logo({ logoWidth }) {
  return <Image src={logo} alt="healthifyme logo" logoWidth={logoWidth} />;
}

Logo.propTypes = {
  logoWidth: PropTypes.string
};

Logo.defaultProps = {
  logoWidth: "40%"
};
