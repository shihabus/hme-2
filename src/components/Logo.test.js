import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";

import Logo from "./Logo";

test("Logo snapshot", () => {
  const { container } = render(<Logo />);
  expect(container.firstChild).toMatchSnapshot();
});
