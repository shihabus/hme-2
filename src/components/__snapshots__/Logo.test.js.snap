// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo snapshot 1`] = `
<img
  alt="healthifyme logo"
  class="sc-AxjAm fqmBvb"
  src="test-file-stub"
/>
`;
